airflow/providers/microsoft/azure/example_dag/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is there a related issue? or open epic? Just to get some context

Reply:
It's related to an issue or epic. There is a general agreement that if there is something (issue or feature) that you have decided to work on, then no need to open an issue for it. Just directly open the PR.

Reply:
Thank you!